packages/pwa/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v1.3.0-dev (Nov 18, 2021)
+
+-   Add message ids to all the translated text, so they provide context for the translators [#239](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/239)
```

Comment:
a small ask: we should also include the other two PRs for messages Ids too. https://github.com/SalesforceCommerceCloud/pwa-kit/pull/207 and https://github.com/SalesforceCommerceCloud/pwa-kit/pull/195